### ISSUE TITLE
Suggest cabal --dependencies-only and interactive mode

### ISSUE THREAD

**Issue by Bodigrim** [chshersh/dr-cabal#30]
I have two suggestions to improve README:

1. `dr-cabal` plots external dependencies only, but not the package itself, so there is little benefit to build the latter. Could we suggest to use `cabal build --dependencies-only`, so that users do not wait for anything else?

3. Shall we advertise interactive mode as the default one, instead of two-step approach with `watch` and `profile`?

**Comment by chshersh** [comment]
As for 1, I agree with the documentation improvement. After #22, `dr-cabal` will be able to track not only dependencies but documentation can be changed only when it's implemented.

As for 2, I've created a separate issue to refactor CLI entirely (and it includes changing the documentation).
